README.md

```diff
@@ -1,2 +1,14 @@
+![image](images/sunglasses.jpg)
+
 # awesome-conda-plugins
-A collection of conda plugins
+
+A curated collection of [conda plugins](https://docs.conda.io/projects/conda/en/latest/dev-guide/plugins/index.html).
+
+## Plugins List
+
+<!-- PLUGIN_LIST -->
+| &nbsp; | &nbsp; | Name  | Version | Description | Author |
+| ----- | ----- | ----- | ----- | ----- | ----- |
+| [🏠](https://github.com/conda/conda-libmamba-solver) | [⬇️](https://anaconda.org/main/conda-libmamba-solver) | `conda-libmamba-solver` | 23.5.0 | A faster conda solver based on the [mamba project](https://mamba.readthedocs.io/en/latest/). | Conda maintainers
+| [🏠](https://github.com/travishathaway/conda-envlock) | [⬇️](https://github.com/travishathaway/conda-envlock#install) | `conda-envlock` | 0.2.0 | Locks and unlocks conda environments to avoid mistakenly modifying them. | Travis Hathaway |
```

Comment:
Let's use the user handle instead:

```suggestion
| [🏠](https://github.com/conda/conda-libmamba-solver) | [⬇️](https://anaconda.org/main/conda-libmamba-solver) | `conda-libmamba-solver` | 23.5.0 | A faster conda solver based on the [mamba project](https://mamba.readthedocs.io/en/latest/). | @conda/conda-libmamba-solver |
| [🏠](https://github.com/travishathaway/conda-envlock) | [⬇️](https://github.com/travishathaway/conda-envlock#install) | `conda-envlock` | 0.2.0 | Locks and unlocks conda environments to avoid mistakenly modifying them. | @travishathaway |
```